.github/workflows/yarf.yaml

```diff
@@ -17,7 +17,33 @@ on:
       - '!.github/workflows/yarf.yaml'
 
 jobs:
-  build:
+  install-jammy:
+    runs-on: [self-hosted, linux, jammy, x64, large]
```

Comment:
I think we are using GitHub-hosted runners moving forward, so this should be `ubuntu-22.04`?

Reply:
github tends to deprecate older versions of ubuntu, and since here being on jammy is important I would stick with the canonical hosted runner